### PR TITLE
Follow User Follower crash fix addressed in #3161.

### PR DESCRIPTION
**Pull request by boldestfortune**

### CODE REVIEW

**Review by boldestfortune — COMMENTED** [review]
changes addressed in #3161, they weren't in the latest update so I thought I would add. This fixes the follow user followers crash for me. Hope this helps.

**Comment by timgrossmann** [comment]
@boldestfortune Thank you for this addition. 
Could you add some logging to both catch statements, please? 
We need to make sure the problems are logged for better debugging.

**Comment by boldestfortune** [comment]
@timgrossmann no problem, happy to help. In terms of logging I'm not entirely sure how to do that. If you want me to copy & paste my a snippet of my log where I encountered the error I can do that. If there is a process for what you are asking for perhaps you can link me to an article/guide?

File: instapy/unfollow_util.py [hunk 1]
```diff
@@ -740,12 +740,17 @@ def dialog_username_extractor(buttons):
...
-                pass  # Element list is too short to have a [1] element
+
+        try:
+            if person and hasattr(person, 'text') and person.text:
+                try:
+                    person_list.append(person.find_element_by_xpath("../../../*")
+                            .find_elements_by_tag_name("a")[1].text)
+
+                except IndexError:
+                    pass  # Element list is too short to have a [1] element
```
Comment (timgrossmann) [inline instapy/unfollow_util.py]: Something like:

```suggestion
                    logger.error("--> person element list is too short to have a [1] element")
                    pass
```

File: instapy/unfollow_util.py [hunk 2]
```diff
@@ -740,12 +740,17 @@ def dialog_username_extractor(buttons):
...
+        except StaleElementReferenceException:
+            pass  # Dont know why this happens, just ignore
```
Comment (timgrossmann) [inline instapy/unfollow_util.py]: Any reason?
```suggestion
            logger.error("--> Don't know why this happens, just ignore")
            pass 
```

Comment (timgrossmann) [reply instapy/unfollow_util.py]: Maybe we should at least describe what happens before the error is thrown

**Comment by boldestfortune** [comment]
Sorry @timgrossmann , still learning alot of this as I go, but here is traceback at crash:

```
OOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO
INFO [2018-10-25 17:45:20] [account]  Session started!
oooooooooooooooooooooooooooooooooooooooooooooooooooooooo


...............................................................
INFO [2018-10-25 17:46:30] [account]  Logged in successfully!
'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''

_____________________________________________________________________________
INFO [2018-10-25 17:46:34] [account]  Starting to follow user `Followers`..
"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
INFO [2018-10-25 17:46:34] [account]  User 'instauser' [1/9]
INFO [2018-10-25 17:47:09] [account]  Sessional Live Report:
        |> LIKED 0 images  |  ALREADY LIKED: 0
        |> COMMENTED on 0 images
        |> FOLLOWED 0 users  |  ALREADY FOLLOWED: 0
        |> UNFOLLOWED 0 users
        |> INAPPROPRIATE images: 0
        |> NOT VALID users: 0



OOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO
INFO [2018-10-25 17:47:09] [account]  Session ended!
oooooooooooooooooooooooooooooooooooooooooooooooooooooo


Traceback (most recent call last):
  File "c:\python37\Lib\pdb.py", line 1697, in main
    pdb._runscript(mainpyfile)
  File "c:\python37\Lib\pdb.py", line 1566, in _runscript
    self.run(statement)
  File "c:\python37\Lib\bdb.py", line 585, in run
    exec(cmd, globals, locals)
  File "<string>", line 1, in <module>
  File "c:\users\user\dev\account\instapy\test.py", line 1, in <module>
    import os
  File "C:\Users\user\dev\account\instapy\instapy\instapy.py", line 2490, in follow_user_followers
    self.logfolder)
  File "C:\Users\user\dev\account\instapy\instapy\unfollow_util.py", line 913, in get_given_user_followers
    channel, jumps, logger, logfolder)
  File "C:\Users\user\dev\account\instapy\instapy\unfollow_util.py", line 720, in get_users_through_dialog
    person_list = dialog_username_extractor(buttons)
  File "C:\Users\user\dev\account\instapy\instapy\unfollow_util.py", line 743, in dialog_username_extractor
    if person and hasattr(person, 'text') and person.text:
  File "C:\Users\user\dev\account\lib\site-packages\selenium\webdriver\remote\webelement.py", line 76, in text
    return self._execute(Command.GET_ELEMENT_TEXT)['value']
  File "C:\Users\user\dev\account\lib\site-packages\selenium\webdriver\remote\webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "C:\Users\user\dev\account\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\user\dev\account\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: element is not attached to the page document
  (Session info: chrome=69.0.3497.100)
  (Driver info: chromedriver=2.40.565498 (ea082db3280dd6843ebfb08a625e3eb905c4f5ab),platform=Windows NT 10.0.17134 x86_64)
```
**and after running script through debugger cause looks to be selenium, not instapy??** :
```

-> raise exception_class(message, screen, stacktrace)

self = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x000001C2CD556F98>
response = {'sessionId': 'a5a67cc55d73b87afe36d098698ba9fc', 'status': 10, 'value': {'message': 'stale element reference: element is not attached to the page document\n  (Session info: chrome=69.0.3497.100)\n  (Driver info: chromedriver=2.40.565498 (ea082db3280dd6843ebfb08a625e3eb905c4f5ab),platform=Windows NT 10.0.17134 x86_64)'}}

237                 if 'data' in value:
238                     alert_text = value['data'].get('text')
239                 elif 'alert' in value:
240                     alert_text = value['alert'].get('text')
241                 raise exception_class(message, screen, stacktrace, alert_text)
242  ->         raise exception_class(message, screen, stacktrace)
243
244         def _value_or_default(self, obj, key, default):
245             return obj[key] if key in obj else default
```

**Comment by timgrossmann** [comment]
@boldestfortune Thank you for adding the logging.
This error most likely is not an issue of your branch, we still need to find a fix for the stale element exception...

Could you make sure that all the checks of CI go through, please?
Click on `details` next to the failed job and check what the problem was.

**Review by boldestfortune — COMMENTED** [review]
Okay @timgrossmann  Sorry about all the failed commits, it was the logger.error function. I know you preferred that in there, but I don't know enough about python to get it to work, and I cant get flake8linter to function at the moment, so I just reverted it until I can figure it out over the weekend.

**Comment by timgrossmann** [comment]
@boldestfortune Sorry for the inconvienices... Could you please create a new PR with the latest version of InstaPy and apply your changes again? 🙈

**Pull request closed without merging**